chart/githubconnector/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
```

Comment:
Same as Application. See: https://helm.sh/docs/chart_best_practices/#resource-naming-in-templates